pbrd/pbr_zebra.c

```diff
@@ -361,7 +361,8 @@ static int pbr_zebra_nexthop_update(int command, struct zclient *zclient,
 	char buf[PREFIX2STR_BUFFER];
 	uint32_t i;
 
-	zapi_nexthop_update_decode(zclient->ibuf, &nhr);
+	if (!zapi_nexthop_update_decode(zclient->ibuf, &nhr))
+		zlog_debug("Failure to decode Nexthop update message");
```

Comment:
I think this should be a `zlog_warn()` and we should return from the function when this error happens.